boot/linkerscript.ld

```diff
@@ -79,176 +77,157 @@ ENTRY( __reset_handler );
 */
 SECTIONS
 {
-.text :
-{
-. = ALIGN(4);
-__text_start = .;
-PROVIDE(__text_start = __text_start);
-. = ALIGN(4);
-KEEP(*(.vectors));
-. = ALIGN(4);
-*(.text .text.* .gnu.linkonce.t.*);
-. = ALIGN(4);
-*(.glue_7t .glue_7);
-. = ALIGN(4);
-*(.rodata .rodata.* .gnu.linkonce.r.*);
-. = ALIGN(4);
-*(.ARM.extab* .gnu.linkonce.armextab.*); /* exception unwinding information */
-. = ALIGN(4);
-*(.gcc_except_table); /* information used for stack unwinding during exception */
-. = ALIGN(4);
-*(.eh_frame_hdr); /* additional information about .ex_frame section */
-. = ALIGN(4);
-*(.eh_frame); /* information used for stack unwinding during exception */
-/*
-
+	.text :
+	{
+		. = ALIGN(4);
+		__text_start = .;
+		KEEP(*(.vectors .vectors.*))
+		*(.text .text.* .gnu.linkonce.t.*); /* text segment */
+		*(.glue_7t .glue_7); /* glue arm to tumb code, glue tumb to arm code*/
+		*(.rodata .rodata.* .gnu.linkonce.r.*); /* Constands, strings, etc */
+		*(.ARM.extab* .gnu.linkonce.armextab.*); /* exception unwinding information */
```

Comment:
Do we need this, since exceptions are disabled?

Reply:
It doesnt look like it. It isn't in the default linkerscript [arm page](http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0474h/BABEHEDA.html). But was in the linker script before and I didn't think of removing it.